docs/diagrams/kedro-no-project.md

```diff
@@ -1,14 +1,14 @@
 ```mermaid
 sequenceDiagram
-    title "$ kedro\ndirectory without Kedro project"
+    title $ kedro in directory without Kedro project
 
-    participant kedro as "$ kedro"
-    participant entrypoint as "setup.py\nkedro = kedro.framework.cli:main"
-    participant init_plugins as "Kedro Plugins\nentry_point = kedro.init"
-    participant kedro_cli as "Kedro CLI\nglobal commands\ninfo, new, docs, starter"
-    participant global_plugins as "Kedro Plugins\nentry_point = kedro.global_commands"
-    participant pyproject.toml as "Current directory\npyproject.toml"
-    participant click as "Click"
+    participant kedro as $ kedro
+    participant entrypoint as pyproject.toml <br> kedro = "kedro.framework.cli:main"
+    participant init_plugins as Kedro Plugins <br> [project.entry-points."kedro.init"]
```

Comment:
Is this a valid entrypoint? I was trying to find info on it but I couldn't.

Reply:
I took me a while to figure this out as well. It's part of the entrypoint groups:

```
``

Reply:
Oh TIL! Thanks!